_episodes/01-introduction.md

```diff
@@ -1,155 +1,127 @@
-- What is the role of the ESMValTool community?
+- What is ESMValTool?
+- Who are the people behind ESMValTool?
 
 objectives:
-- Describe key strengths and weaknesses of ESMValTool
-- Know the different components of ESMValTool
-- Understand the concept of a community-driven software project
+- Familiarize with ESMValTool
+- Synchronize expectations
 
 keypoints:
 - ESMValTool provides a reliable interface to analyse and evaluate climate data
-- Using ESMValTool promotes standardization, collaboration, and reuse
+- A large collection of recipes and diagnostic scripts is already available
 - ESMValTool is built and maintained by an active community of scientists and
   developers
-- ESMValTool is written in Python, but supports diagnostic scripts in multiple
-  languages
-
 ---
 
 ## What is ESMValTool?
 
-ESMValTool is first and foremost a tool to analyse climate data. But you
-probably already knew that and we like to think there's more to it than that. So
-let's start with a quick check to synchronize our expectations.
+This tutorial is a first introduction to ESMValTool. Before diving into the
+technical steps, let's talk about what ESMValTool is all about.
 
-> ## Question: what is ESMValTool
->
-> Which of the following items would you say apply to ESMValTool?
+> ## What is ESMValTool?
 >
-> - A tool to analyse climate data
-> - The easy way out
-> - A community effort
-> - Free
-> - A command line tool
-> - A way to make climate science more [FAIR](https://fair-software.eu/about)
-> - Perfect
-> - Suitable for (Jupyter) notebooks
+> What do you already know about, or expect from ESMValTool?
 >
-> Check our answers by unfolding the box below.
->
-> > ## ESMValTool is
-> >
-> > &#10003; **A tool to analyse climate data.**  It takes care of finding,
-> > opening, checking, fixing, concatenating, and preprocessing CMIP data and
-> > several other supported datasets.
+> > ## ESMValTool is...
 > >
-> > &#10005;  **The easy way out.** If you just want to do an exploratory
-> > analysis or quickly hack something together, ESMValTool is probably not the
-> > way to go. The tool is intended for robust, repeatable and shareable climate
-> > analysis. That *does* require a bit more effort.
+> > EMSValTool is many things, but in this tutorial we will focus on the
+> > following traits:
 > >
-> > &#10003; **A community effort.** ESMValTool is developed and maintained by a
-> > large team of climate scientists and software engineers. It is an open
-> > source project to which anyone can contribute. Its longevity depends on
-> > these contributions.
+> > &#10003; **A tool to analyse climate data**
 > >
-> > &#10003; **Free.** ESMValTool is licenced under Apache 2.0, which means
-> > everyone can use, modify, or share it free of charge. However, we *do*
-> > encourage all users to contribute to the community once they get more
-> > comfortable with the tool.
+> > &#10003; **A library for reproducible climate science**
```

Comment:
The term "library" is really confusing here. A lot of people will think "software library" while it is primarily used as an application, the API is not the primary entrypoint into ESMValTool.

"A collection of diagnostics for reproducible climate science"

This is still not perfect, but will lead to less thinking in the lines of:  "ahh, I see, its a Python library like Iris and Xarray"